### PR TITLE
Updated .compute

### PR DESCRIPTION
**Pull request by tilmankamp**
X-DeepSpeech: NOBUILD

### CODE REVIEW

File: .compute [hunk 1]
```diff
@@ -2,12 +2,14 @@
 
 set -xe
 
-apt-get install -y python3-venv
+apt-get install -y python3-venv libopus0
```
Comment (kdavis-mozilla) [inline .compute]: Why isn't libopus0 in requirements.txt?

File: .compute [hunk 2]
```diff
@@ -2,12 +2,14 @@
...
+
 python3 -m venv /tmp/venv
 source /tmp/venv/bin/activate
 
+pip install -U setuptools wheel pip
 pip install -r <(grep -v tensorflow requirements.txt)
-pip install tensorflow-gpu==1.15.0
+pip install tensorflow-gpu==1.14
```
Comment (kdavis-mozilla) [inline .compute]: We're now on 1.15.0

File: .compute [hunk 3]
```diff
@@ -18,17 +20,23 @@ data="${SHARED_DIR}/data"
...
   --train_batch_size 24 \
   --dev_batch_size 48 \
   --test_batch_size 48 \
+  --train_cudnn \
   --n_hidden 2048 \
   --learning_rate 0.0001 \
-  --dropout_rate 0.2 \
-  --epoch 13 \
+  --dropout_rate 0.40 \
+  --epochs 200 \
```
Comment (kdavis-mozilla) [inline .compute]: Are this many epochs needed? I've found that after about 125-150 it's already converged and a new run, continuing the first, with a lower learning rate is required

File: .compute [hunk 4]
```diff
@@ -18,17 +20,23 @@ data="${SHARED_DIR}/data"
...
+  --noearly_stop \
+  --feature_cache "../tmp/feature.cache" \
   --checkpoint_dir "../keep" \
   --summary_dir "../keep/summaries"
+
```
Comment (kdavis-mozilla) [inline .compute]: nit no extra new line needed

File: .compute (hunk 1 above)
Comment (tilmankamp) [reply .compute]: This is the system-level Opus library which is required by the Python package.
The wrapping module in `requirements.txt`:
```opuslib == 2.0.0```

File: .compute (hunk 2 above)
Comment (tilmankamp) [reply .compute]: Because our cluster's CUDA version does not match.

File: .compute (hunk 3 above)
Comment (tilmankamp) [reply .compute]: Will set it to 150 then.

File: .compute (hunk 1 above)
Comment (kdavis-mozilla) [reply .compute]: Oh I'm dumb I was reading this is a pip install.

File: .compute (hunk 2 above)
Comment (kdavis-mozilla) [reply .compute]: Where are the CUDA requirements for TF 1.15? I don't see them [here](https://www.tensorflow.org/install/source#linux).

Also, I know I've asked this before, but what CUDA version is our server at?

Comment (reuben) [reply .compute]: FWIW, I've been using TF 1.15 on the cluster for a long time now. It works fine.

Comment (reuben) [reply .compute]: Also, using an older version for training models after we upgraded the native client to use 1.15 can lead to unexpected behavior at the native client side. We should stick to using the same version on both sides.

Comment (tilmankamp) [reply .compute]: Could have been a problem with my own machine. Just started a test run to verify that 1.15 is actually working on the cluster.
